src/db/index.js

```diff
@@ -281,6 +288,11 @@ module.exports = function(formio) {
         config.mongoFeatures.compoundIndexWithNestedPath = formio.mongoFeatures.compoundIndexWithNestedPath = false;
       }
       await featuresTest.drop();
+    }
+    catch (err) {
+      formio.util.log('Error determining MongoDB compatibility:');
+      formio.util.log(err);
+    }
```

Comment:
If this _were_ to fail, the worst that could happen is a tiny extra collection, so I added this try to catch to avoid an all out crash just in case